test-suite/output/PrintNotation.out

```diff
@@ -0,0 +1,129 @@
+at level 80, right associativity.
+Notation "_ \/ _" at level 85 with arguments constr at next level, constr
+at level 85, right associativity.
+Notation "~ _" at level 75 with arguments constr at level 75,
+right associativity.
+Notation "_ = _ :> _" at level 70 with arguments constr at next level, constr
+at next level, constr at next level, No associativity defined.
+Notation "_ = _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ = _ = _" at level 70 with arguments constr at next level, constr
+at next level, constr at next level, No associativity defined.
+Notation "_ <> _ :> _" at level 70 with arguments constr at next level,
+constr at next level, constr at next level, No associativity defined.
+Notation "_ <> _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ <= _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ < _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ >= _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ > _" at level 70 with arguments constr at next level, constr
+at next level, No associativity defined.
+Notation "_ <= _ <= _" at level 70 with arguments constr at next level,
+constr at next level, constr at next level, No associativity defined.
+Notation "_ <= _ < _" at level 70 with arguments constr at next level, constr
+at next level, constr at next level, No associativity defined.
+Notation "_ < _ < _" at level 70 with arguments constr at next level, constr
+at next level, constr at next level, No associativity defined.
+Notation "_ < _ <= _" at level 70 with arguments constr at next level, constr
+at next level, constr at next level, No associativity defined.
+Notation "_ + _" at level 50 with arguments constr at level 50, constr
+at next level, left associativity.
+Notation "_ - _" at level 50 with arguments constr at level 50, constr
+at next level, left associativity.
+Notation "_ * _" at level 40 with arguments constr at level 40, constr
+at next level, left associativity.
+Notation "_ / _" at level 40 with arguments constr at level 40, constr
+at next level, left associativity.
+Notation "- _" at level 35 with arguments constr at level 35,
+right associativity.
+Notation "/ _" at level 35 with arguments constr at level 35,
+right associativity.
+Notation "_ ^ _" at level 30 with arguments constr at next level, constr
+at level 30, right associativity.
+Notation "_ || _" at level 50 with arguments constr at level 50, constr
+at next level, left associativity.
+Notation "_ && _" at level 40 with arguments constr at level 40, constr
+at next level, left associativity.
+Notation "( _ , _ , .. , _ )" at level 0 with arguments constr, constr,
+No associativity defined.
```

Comment:
@jfehrle Here is an example of a recursive notation.

Reply:
What does it display?

Reply:
It displays that, this is the output. The command was `  Print Notation "( x , y , .. , z )".`